app/lib/report_helper.rb

```diff
@@ -503,7 +503,8 @@ def self.cleanup_genus_ids!(taxon_counts_2d)
 
   def self.validate_names!(tax_2d)
     # This converts superkingdom_id to category_name and makes up
-    # suitable names for missing and blacklisted genera and species.
+    # suitable names for missing and blacklisted genera and species. Such
+    # made-up names should be lowercase so they are sorted below proper names.
```

Comment:
This seems justifiable in general because 

a) it is easy to uppercase downstream
b) it is consistent with missing values defined elsewhere such as "uncultured bacterium"